guides/V1_get_space_template.md

```diff
@@ -0,0 +1,69 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'get-space-template'
+is: 'experimental'
+---
+
+# Get Space Template
+
+A user can fetch details of a space template by passing its id to the _spaceTemplate_ query, provided the user either created
+the template or is subscribed to the offering with which the template is associated.
+
+## Schema
+
+### Fetch A Space Template
+
+```graphql
+type QueryRoot {
+  ...
+  spaceTemplate(id: ID!): SpaceTemplate
+}
+```
+
+## Example request
+
+~~~~
+Method: POST
+URL: https://api.watsonwork.ibm.com/graphql
+Headers: 'Content-Type: application/graphql' , 'x-graphql-view: PUBLIC, EXPERIMENTAL'
+Body:
+{
+  spaceTemplate(id: "template-id") {
+    name
+    requiredApps {
+      items {
+        id
+      }
+    }
+    properties {
+      items {
+        id
+        type
+        displayName
+        ... on SpaceListProperty {
+          defaultValue
+          acceptableValues {
+            id
+            displayName
+          }
+        }
+        ... on SpaceTextProperty {
+          defaultValue
+        }
+        ... on SpaceBooleanProperty {
+          defaultStringValue
+        }
+      }
+    }
+    spaceStatus {
+      defaultValue
+      acceptableValues {
+        id
+        displayName
+      }
+    }
+  }
+}
+~~~~
+
```

Comment:
Would it make sense to have a sample output?

Reply:
Added.

Reply:
I think we should refer to the explorer in this case. The explorer is more useful than the documentation for specific examples. Outputs should be a 1:1 mapping to the request fields and it is more costly to maintain the type structures in documentation also, whereas the explorer will always be up to date.

Reply:
Makes sense to me. I'm open to whatever is the consensus view here.

Reply:
Replaced sample output from `spaceTemplate` with link to explorer.